<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>859</width>
    <height>620</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>841</width>
      <height>561</height>
     </rect>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Rounded</enum>
    </property>
    <property name="currentIndex">
     <number>3</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Общее</string>
     </attribute>
     <widget class="QPushButton" name="stateButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>100</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>PushButton</string>
      </property>
      <property name="default">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="firewallWork">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>15</y>
        <width>281</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Работа фаервола:</string>
      </property>
     </widget>
     <widget class="QLabel" name="lastActions">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>20</y>
        <width>141</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Последние действия</string>
      </property>
     </widget>
     <widget class="QListWidget" name="recentActionsListWidget">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>40</y>
        <width>351</width>
        <height>192</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Правила</string>
     </attribute>
     <widget class="QTableWidget" name="listOfRules">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>811</width>
        <height>341</height>
       </rect>
      </property>
      <attribute name="horizontalHeaderCascadingSectionResizes">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>100</number>
      </attribute>
      <column>
       <property name="text">
        <string>Номер</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Вкл/выкл</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Действие</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Протокол</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Источник</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Порт источника</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Порт</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Порт назначения</string>
       </property>
      </column>
     </widget>
     <widget class="QPushButton" name="deleteRuleButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>410</y>
        <width>121</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Удалить правило</string>
      </property>
     </widget>
     <widget class="QPushButton" name="addeditRuleButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>370</y>
        <width>231</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Добавить/редактировать правило</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Подключения</string>
     </attribute>
     <widget class="QTableWidget" name="listOfConnections">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>602</width>
        <height>371</height>
       </rect>
      </property>
      <attribute name="horizontalHeaderMinimumSectionSize">
       <number>24</number>
      </attribute>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>150</number>
      </attribute>
      <attribute name="verticalHeaderMinimumSectionSize">
       <number>25</number>
      </attribute>
      <attribute name="verticalHeaderDefaultSectionSize">
       <number>35</number>
      </attribute>
      <column>
       <property name="text">
        <string>Локальный IP:Порт</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Удаленный IP:Порт</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Протокол</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Состояние</string>
       </property>
      </column>
     </widget>
     <widget class="QPushButton" name="terminateConnectionButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>390</y>
        <width>171</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Разорвать соединение</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Логи</string>
     </attribute>
     <widget class="QCheckBox" name="approvedPackets">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>271</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Добавлять в логи одобренные пакеты</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="prohibitedPackets">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>281</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Добавлять в логи запрещенные пакеты</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="rejectedPackets">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>100</y>
        <width>271</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Добавлять в логи отклоненные пакеты</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="tcpPackets">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>140</y>
        <width>311</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Добавлять в логи пакеты с протоколом TCP</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="udpPackets">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>180</y>
        <width>311</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Добавлять в логи пакеты с протоколом UDP</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="icmpPackets">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>220</y>
        <width>311</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Добавлять в логи пакеты с протоколом ICMP</string>
      </property>
     </widget>
     <widget class="QPushButton" name="addIpButton">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>260</y>
        <width>100</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Добавить IP</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>260</y>
        <width>113</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>310</y>
        <width>113</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="addPortButton">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>310</y>
        <width>111</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Добавить порт</string>
      </property>
     </widget>
     <widget class="QLabel" name="unloggedIP">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>20</y>
        <width>171</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Список нелоггируемых IP</string>
      </property>
     </widget>
     <widget class="QLabel" name="unloggedPorts">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>270</y>
        <width>201</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Список нелоггируемых портов</string>
      </property>
     </widget>
     <widget class="QPushButton" name="chooseFolderButton">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>390</y>
        <width>111</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Выбрать папку</string>
      </property>
     </widget>
     <widget class="QListWidget" name="unloggedIPListWidget">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>50</y>
        <width>256</width>
        <height>192</height>
       </rect>
      </property>
     </widget>
     <widget class="QListWidget" name="unloggedPortListWidget">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>300</y>
        <width>256</width>
        <height>192</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="removeIpButton">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>260</y>
        <width>100</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Удалить IP</string>
      </property>
     </widget>
     <widget class="QPushButton" name="removePortButton">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>310</y>
        <width>100</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Удалить порт</string>
      </property>
     </widget>
     <widget class="QLabel" name="logFolderPathLabel">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>360</y>
        <width>391</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>859</width>
     <height>37</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
